packages/prices-api/src/llamalend/parsers.ts

```diff
@@ -1,41 +1,48 @@
-import { toDate } from '../timestamp'
-import type * as Responses from './responses'
-import type * as Models from './models'
-
 export const parseMarket = (x: Responses.GetMarketsResponse['data'][number]): Models.Market => ({
   name: x.name,
   controller: x.controller,
   vault: x.vault,
   llamma: x.llamma,
   policy: x.policy,
   oracle: x.oracle,
-  rate: parseFloat(x.rate),
-  apyBorrow: parseFloat(x.borrow_apy),
-  apyLend: parseFloat(x.lend_apy),
+  oraclePools: x.oracle_pools,
```

Comment:
@0xAlunara it looks like the server sends numbers not strings. Also noticed some fields were missing.